### PR TITLE
Implement nearMedia filters for Aggregate query

### PR DESCRIPTION
**Pull request by parkerduckworth**

### CODE REVIEW

File: v4/test/graphql/graphql_test_helper.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package graphql
```
Comment (antas-marcin) [inline v4/test/graphql/graphql_test_helper.go:1]: shouldn't the file be named `graphql_helper_test.go`? are we planning to use this file outside tests?

Comment (parkerduckworth) [reply v4/test/graphql/graphql_test_helper.go]: fixed

File: v4/weaviate/graphql/aggregatequerybuilder.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
+package graphql
```
Comment (antas-marcin) [inline v4/weaviate/graphql/aggregatequerybuilder.go:1]: I would leave the name `aggregate.go` I know that you wanted to align this change with `getquerybuilder.go` but for get we have `get.go` that uses `getquerybuilder.go` and that's why we have the name for this like `getquerybuilder.go` I think that we can leave the `aggregate.go`

File: v4/weaviate/graphql/nearvectorbuilder.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	return b
+}
+
+// Build build the given clause
+func (b *NearVectorArgumentBuilder) build() string {
+	clause := []string{}
+	if b.withCertainty {
+		clause = append(clause, fmt.Sprintf("certainty: %v", b.certainty))
+	}
+	if b.vector != nil {
```
[the remark this answers is not in the document]
Comment (parkerduckworth) [reply v4/weaviate/graphql/nearvectorbuilder.go]: done

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**